site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java

```diff
@@ -271,6 +282,35 @@ private HstQuery constructQuery(HstQueryBuilder queryBuilder, Constraint searchS
     /**
      * Publication System content types that should be included in search
      */
+    private void deriveTypes(final HstQueryBuilder query, final HstRequest request) {
```

Comment:
Just FYI I think I'm going to remove this and replace it with one single list of all the doctypes (RPS + NIL + DW) and then let the folder(/s) be the single source of filtering for the tabs. Happy to merge like this and I'll fix it later. Have discussed with @wojtek-oledzki as I know he told you the opposite originally 😉

Reply:
I am pushing the changes for DW-223 directly in this pull request, I think it's easier to sort it out here. I will update this PR shortly

Reply:
👍